website/docs/kubernetes/configuring-editing-kube-object.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 12
+title: Configuring or editing an object
+description: Configuring or editing a kubernetes object
+keywords: [podman desktop, podman, deploying, images, kubernetes]
+tags: [configuring-kubernetes, images]
+---
+
+# Configuring or editing a Kubernetes object
```

Comment:
on the wording, referring to https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-command/

maybe it's Managing and updating rather than configuring and editing

Reply:
Agree. let's change the heading to `Managing Kubernetes objects` because manage covers all the aspects create, update, delete, and other actions.